src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -254,6 +254,13 @@ void otSysSetInfraNetif(const char *aInfraNetifName, int aIcmp6Socket);
  */
 bool otSysInfraIfIsRunning(void);
 
+/**
+ * Initialize CLI module for Daemon on the system.
```

Comment:
```suggestion
 * Initializes the CLI module using the daemon.
```
May also specify the behavior in a new paragraph: the daemon will handle the CLI output.

Reply:
Done.